src/components/molecules/SetSelectorGroup/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './SetSelectorGroup.js'
```

Comment:
@marinagustint Remove .js from Here

This should be just
`export { default } from './SetSelectorGroup'`